docker-compose.dev.yml

```diff
@@ -36,6 +36,7 @@ services:
       - DTFS_CENTRAL_API_KEY
       - EXTERNAL_API_KEY
       - NODE_ENV=${NODE_ENV}
+      - RATE_LIMIT_THRESHOLD
```

Comment:
@abhi-markan we'll need to add these environment variables to all our GHAs and deployment scripts